lang/lang_en_fr.txt

```diff
@@ -234,10 +262,26 @@
 "Error - static memory has been overwritten"
 "Erreur - la memoire statique a ete ecrasee"
 
+#MSG_CUT_FILAMENT c=16
+"Cut filament"
+"Coupe filament"
+
```

Comment:
cut filament =coupure filament

Reply:
or "couper filament"

Reply:
I believe that `coupure filament` translates to "a break in the filament" or "filament cut"(a cut made by filament). I believe that `Coupe` (imperative conjugation of "Couper"-to cut) `filament` is correct

Reply:
or "Filament coupe" ( filament coupé)

Reply:
> cut filament =couper filament

I believe that `Couper` should be conjugated to `Coupez`(more polite) or `Coupe`(what we use for most translations).

Reply:
Stays as I translated it.